src/libs/state.ts

```diff
@@ -17,7 +18,7 @@ export const authStore = Utils.atom({
   isAzurePreviewUser: undefined,
 });
 
-export const getUser = () => authStore.get().user;
+export const getUser: User = () => authStore.get().user;
```

Comment:
See comment above -- I don't think this is actually returning a `oidc-client-ts/User` object.